nxdrive/engine/watcher/remote_watcher.py

```diff
@@ -757,7 +757,6 @@ def _update_remote_states(self) -> None:
                     f"Skipping unsyncable document {change} (digest is 'notInBinaryStore')"
                 )
                 continue
-
```

Comment:
Keep it. After a `return` or a `return` we should keep/add a blank line to separate things. And ease reading too.